packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -92,7 +92,7 @@ function BaseDrawer({
     <div
       className={classNames(
         'fixed inset-0 transition-opacity duration-300 ease-in-out',
-        isFullScreen ? 'z-modal' : 'z-max bg-overlay-quaternary-onion',
+        isFullScreen ? 'z-modal' : 'z-modal bg-overlay-quaternary-onion',
```

Comment:
`z-modal` is used in both cases. I think we can add `z-modal` to the line above, and change this one to achieve the same functionality. Seems shorter to me:
```suggestion
        !isFullScreen && 'bg-overlay-quaternary-onion',
```

Reply:
Right. Yes, this is definitely better.